lib/facter/self_service.rb

```diff
@@ -108,6 +108,13 @@
     { S0013: File.open(summary_path).read.include?('catalog_application') }
   end
 
+  chunk(:S0019) do
+    next unless PuppetSelfService.primary? || PuppetSelfService.replica? || PuppetSelfService.compiler? || PuppetSelfService.legacy_compiler?
+    pupserv_api = PuppetSelfService.status_check('8140', 'pe-jruby-metrics?level=debug')
+    jruby_check = pupserv_api['status']['experimental']['metrics']['average-free-jrubies']
+    { S0019: jruby_check > 1 }
```

Comment:
Looks like it failed on a few still. When I was testing earlier I also checked what the API was returning if it reported `false` and saw it would get as low as `0.84` after a Puppet run.